### ISSUE TITLE
Rethink if/how to round numbers

### ISSUE THREAD

**Issue by jellegerbrandy** [daostack/alchemy#662]
Sorry but I need to put this on the agenda again. So staking is inteded for traders, and they need PRECISE and predictable information. 

For example, in this scenario, **XX GEN to boost is frustratingly incorrect**:

- stake 1 for fail, bringing to total of fail to `1+e`
- see the notification `16GEN needed to boost`
- stake 16 gen on pass
- see that nothing happens, bc you need to stake 16+e gen

there are many other cases where information is just hard to parse. I have now proposals with the same staking fail/pass staking amouts, but one says `0.12 needed to boost` and the other `0.13 needed to boost`. This is just unexplainable.

**I think we should stop rounding stuff and just show everythign behing the comma, just as trading platforms do**

**Comment by jellegerbrandy** [comment]
Similarly, seeing `+0 GEN to un-boost` or similar messages are frustrating as well, because you have no way of knowing how much REALLY.

**Comment by dkent600** [comment]
You can do an hover that shows the entire amount with a button that copies it to the clipboard.

I'm not sure about hover for UX on mobile devices (mobile has to matter for Alchemy eventually).  So an alternative could be a small button, placed next to any amount that has been truncated, that pops up a control that shows the entire amount with a button that copies it to the clipboard.

**Comment by jellegerbrandy** [comment]
rethought it was :-)
